options/dopex/clamm.ts

```diff
@@ -74,23 +77,23 @@ async function getChainStats({ graphUrl, timestamp }: IGetChainStatsParams) {
     (acc, market) => {
       return {
         dailyNotionalVolume: acc.dailyNotionalVolume + Number(market.volume),
-        dailyPremiumVolume: acc.dailyPremiumVolume + Number(market.premium),
+        dailyFees: acc.dailyFees + Number(market.premium),
```

Comment:
for options we can't include premium on fees

Reply:
Oh okay! How can we show premiums as premiums and not as premium volume (which is only visible on a graph). premiums should be shown in the side bars as well along fees, revenue etc.

Reply:
<img width="1220" alt="image" src="https://github.com/DefiLlama/dimension-adapters/assets/11234402/d2ba8a7c-67ce-4db7-a6e8-e63e637bd3d6">
you mean rev 24h, it is wrong file is should be 24h premiums data we will fix

Reply:
![image](https://github.com/DefiLlama/dimension-adapters/assets/90272722/abf57728-1a4b-4b3b-9856-505d22ce8d23)

premiums should show here on the left panel too

Reply:
> for options we can't include premium on fees

Hey, your docs mention: 
<img width="784" alt="Screenshot 2024-03-09 at 10 15 43 PM" src="https://github.com/DefiLlama/dimension-adapters/assets/79846912/f49d9db3-8c8f-4e15-8c21-1978b8977571">

Which means premiums should be included in fees since all premiums are earned directly by our LPs on top of the fees they earn from Uniswap V3.

Reply:
no fees is only add on fees when user buy option premium is not fees

Reply:
> Yes, 24h revenue should show as premiums 24h. anyways I've updated the code to show only revenue and not fees since they're the same data :)

if you need update fees/rev plz set at field `dailyFees`, `dailyRevenue`
now current adapter set fees = rev then show fees = rev premiums is show only options page

Reply:
If that's the case, no change has to be made and this PR can be closed. Our only concern is premiums not showing in main options page (image shown in my last message) which as you mentioned will be fixed :)

Reply:
yes will take look why option is show daily rev instead daily premium

Reply:
Hey! updated the PR to just show more stats as per teams feedback :) please review